Maps.md

```diff
@@ -0,0 +1,21 @@
+This is a general guide about Maps for all FullScreenShenanigans projects. This guide will explain the concept of Maps, Areas, and Locations within a game and how they're connected.
+
+### Table of Contents
+1. [MapsCreatr](#mapscreatr)
+2. [AreaSpawnr](#areaspawnr)
+
+# MapsCreatr
+
+MapsCreatr is a module that handles the storage of Maps, Areas, and Locations. 
+
+A Map is a collection of Areas. An Area is a 2D space with predefined  `x` and `y` boundaries and Things. Locations are entry points within an Area, where an entry point is where the player may end up after transitioning to a new Area.
+
+Generally in a FullScreenShenanigans project, each Map information is stored in its own .js file, outlining all Areas and Locations within that map. When the game is started, MapsCreatr loads all this data, goes through every Map's raw JSON data to create a new Map object using ObjectMakr, and finally storing it for later reference.
+
+More can be read about MapsCreatr on its [Readme](https://github.com/FullScreenShenanigans/MapsCreatr/blob/master/README.md).
+
+# AreaSpawnr
+
+AreaSpawnr is a module that works with MapsCreatr to spawn Things in a certain Area.
+
+When a new Area is entered by the player, AreaSpawnr uses the information from that Area's Map, stored in MapsCreatr, to spawn that Area's predefined Things (or PreThings). These are spawned in the Area, but only the Things in the current viewable area, or bounding box. The PreThing's `x` and `y` positioning are also defined in the Map's .js file and loaded into MapsCreatr.
```

Comment:
> also defined in the Map's .js file

Two problems:
1. "also" defined? Where else?
2. Specifically where in the file?